src/WebJobs.Script.Analyzers/Constants.cs

```diff
@@ -14,6 +14,7 @@ internal static class DiagnosticsCategories
         {
             public const string Reliability = "Reliability";
             public const string Usage = "Usage";
+            public const string WebJobs = "WebJobs";    // TODO: Too vague?
```

Comment:
What's the usage of this and where are you trying to represent? Context might help me answer that "todo" question :D

Reply:
This is a fun discussion opportunity. From what I've gleaned, we can define categories for our analyzers. Here's an example from Roslyn: https://github.com/dotnet/roslyn-analyzers/blob/main/src/Utilities/Compiler/DiagnosticCategory.cs

You can use categories to configure settings, ex. severity, suppression, etc: https://docs.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2022#set-rule-severity-of-multiple-analyzer-rules-at-once-in-an-editorconfig-file

Since we only have a few analyzers right now, I'm not sure how we want to divide them + future ones up. Since these analyzers operate on the WebJobs binding model, so not fully "Functions", I went with `WebJobs`, but also considered `Binding` as a possible name. Love to hear your thoughts here! + @fabiocav

Reply:
Ahh I see, would `WebJobsBindings` be a good category name?